.taskcluster.yml

```diff
@@ -27,6 +27,7 @@ tasks:
                 event.ref == "refs/heads/master": [{name: firefox, channel: nightly}, {name: chrome, channel: dev}],
                 event.ref == "refs/heads/epochs/daily": [{name: firefox, channel: stable}, {name: chrome, channel: stable}, {name: webkitgtk_minibrowser, channel: nightly}],
                 event.ref == "refs/heads/epochs/weekly": [{name: firefox, channel: beta}, {name: chrome, channel: beta}, {name: webkitgtk_minibrowser, channel: stable}],
+                event.ref == "refs/heads/epochs/three_hourly": [{name: firefox, channel: stable}, {name: chrome, channel: stable}, {name: webkitgtk_minibrowser, channel: nightly}],
```

Comment:
Is this change for debugging? In the event that it sticks around, can you remove the epochs/daily trigger? Otherwise we'll get two runs if this is still the state of the repo next URT midnight.

Reply:
Is for testing faster, I intent to remove the three_hourly trigger as soon as the daily works as expected.